sharding-proxy/src/main/java/io/shardingsphere/shardingproxy/frontend/ShardingProxy.java

```diff
@@ -87,6 +109,11 @@ public void start(final int port) throws InterruptedException {
         }
     }
     
+    private ExecutorService createCommandExecutorService() {
+        int nThreads = GLOBAL_REGISTRY.getShardingProperties().<Integer>getValue(ShardingPropertiesConstant.ACCEPTOR_SIZE);
+        return new ThreadPoolExecutor(nThreads, nThreads, 0, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<Runnable>(16));
```

Comment:
we should use excecuteService like ShardingExecuteEngine way, also make blockingQueue was fixed at 16 is not a good idea.

Reply:
It's a good idea to use ShardingExecuteEngine, I will follow the suggestion.